public/javascripts/settingsController.js

```diff
@@ -14,6 +14,14 @@ var settingsController = function($scope, $http, $location) {
     });
   };
 
+  $scope.delUser = function(username, userArray, delUserIdx) {
+    $http.post('api/delUser/'+username._id).then(function successCallback(response) {
+      userArray.splice(delUserIdx, 1);
```

Comment:
The callback for findOneAndRemove returns the deleteled object. We'll have to make another GET request to get all the remaining users in the callback. I think doing splice is nicer.